nodes/sick_viz.cpp

```diff
@@ -31,8 +32,28 @@ SafetyFieldVisualizer::SafetyFieldVisualizer(const std::string& robot, const std
 
     preprocessFieldData();
 
-    // Subscribe to the active monitoring case topic
     raw_data_sub_ = nh_.subscribe("/" + robot_ + "/" + laser_ + "_nanoscan/output_paths", 1, &SafetyFieldVisualizer::microscanCallback, this);
+    dyn_reconf_sub_ = nh_.subscribe("/" + robot_ + "/" + laser_ + "_nanoscan/parameter_updates", 1, &SafetyFieldVisualizer::dynamicReconfigCallback, this);
```

Comment:
Regularly I would agree and set up another instance of `dynamic_reconfigure::Server` for this node, but in this instance I believe it makes more sense to piggy back off the dyn reconfig server set up by the sick driver. The reasons were 1. to keep all the reconfigurable parameters for each lidar under the same server and 2. to avoid setting up 6 reconfig servers for each fieldset (protective, dtz, protective, dtz, etc.). So it's still kept in a param server, just defined in sick_safetyscanners, not sick_viz. All the perks will still be there with a bit cleaner of a setup.